src/main/java/teammates/sqllogic/api/Logic.java

```diff
@@ -95,6 +95,15 @@ public AccountRequest createAccountRequest(String name, String email, String ins
         return accountRequestLogic.createAccountRequest(name, email, institute, status, comments);
     }
 
+    /**
+     * Gets the account request with the given {@code id}.
+     *
+     * @return account request with the given {@code id}.
+     */
+    public AccountRequest getAccountRequest(UUID id) {
+        return accountRequestLogic.getAccountRequest(id);
+    }
+
     /**
      * Gets the account request with the given email and institute.
```

Comment:
Shall we also remove the old methods (and update places of usage) as well?

Reply:
Yes, this is only the first PR. There's more that needs to be done. I'm doing this first, because after this, then I think the other usages can then probably be updated in parallel.

Reply:
Sure, let's merge this first